docs/docs/external-volumes

```diff
@@ -0,0 +1,81 @@
+--------------------------------------------------------------
+title: Stateful Applications Using Persistent External Volumes
+--------------------------------------------------------------
+
+# Stateful Applications Using Persistent External Volumes
+
+<div class="alert alert-danger" role="alert">
+  <span class="glyphicon glyphicon-exclamation-sign" aria-hidden="true"></span> Adapted in Marathon Version 1.0 <br/>
+  The Persistent Storage functionality is considered beta, so use this feature at your own risk. We might add, change, or delete any functionality described in this document.
+</div>
+
+Marathon applications lose their state when they terminate and are relaunched. In some contexts, for instance, if your application uses MySQL, you’ll want your application to preserve its state. You can use an external storage service, such as Amazon's Elastic Block Store (EBS), to create a persistent volume that follows your application instance.
+
+Using an external storage service allows your apps to be more fault-tolerant. If a host fails, Marathon reschedules it on another host, along with its associated data, without user intervention.
+
+# Specifying an External Volume
+
+When you install DCOS, add the following to your `genconf.yml` file:
```

Comment:
When we merge the Marathon docs with the rest at docs.mesosphere.com, that won't be an issue. For now, I'll check with @air. And, if I understand right, you need to be using DCOS to use external volumes?